pennylane/ops/qubit/parametric_ops.py

```diff
@@ -1269,6 +1269,182 @@ def control_wires(self):
         return self.wires[0:1]
 
 
+class PCPhase(Operation):
+    r"""
+    A Projector Controlled Phase gate.
+
+    .. math:: \Pi_{\phi}(\phi, dim=2, wires=[0,1]) = \begin{bmatrix}
+                e^{i\phi} & 0 & 0 & 0 \\
+                0 & e^{i\phi} & 0 & 0 \\
+                0 & 0 & 1 & 0 \\
+                0 & 0 & 0 & 1
+            \end{bmatrix}.
+
+    Args:
+        phi (float): rotation angle :math:`\phi`
+        dim (int): the qubit subspace on which we apply the phase shift
+        wires (Any, Wires): the wire the operation acts on
+        do_queue (bool): Indicates whether the operator should be
+            immediately pushed into the Operator queue (optional)
+        id (str or None): String representing the operation (optional)
+
+    **Details:**
```

Comment:
Does Details normally come before or after Args?

Reply:
According to this doc: https://docs.pennylane.ai/en/stable/development/guide/documentation.html it comes after. However, for the `PhaseShift` gate I have seen it come before. Let me know what you think we should set as the standard and I can stick with that!

Reply:
For all of the parametric ops it seems to come first, I will swap to that standard